examples/1-execute-query-mutation/webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     alias: {
       react: path.resolve("./node_modules/react"),
       "react-dom": path.resolve("./node_modules/react-dom"),
-      urql: path.resolve("./node_modules/urql"),
+      urql: path.resolve("../../node_modules/urql"),
```

Comment:
This wasn't being consistently resolved in examples using symlinking, so I decided to always resolve at root `node_modules`.